docs/api/boligsameie.md

```diff
@@ -24,13 +24,36 @@ For generell informasjon om tjenestene se egne sider om:
 * [Teknisk spesifikasjon](../om/tekniskspesifikasjon.md)
 
 ## Scope
-Følgende scope skal benyttes ved autentisering i Maskinporten: `skatteetaten:bolisameie`
+Følgende scope skal benyttes ved autentisering i Maskinporten: `skatteetaten:boligsameie`
 
 ## Delegering
-Tilgang til dette API-et kan delegeres i Altinn, f.eks. dersom leverandør benyttes for den tekniske oppkoblingen. Søk opp følgende tjeneste i Altinn for å delegere tilgangen: `Bolisameie API - På vegne av`
+Tilgang til dette API-et kan delegeres i Altinn, f.eks. dersom leverandør benyttes for den tekniske oppkoblingen. 
+Søk opp følgende tjeneste i Altinn for å delegere tilgangen: `Boligsameie API - På vegne av`
 
 ## Teknisk spesifikasjon
-URL-er til API-et, beskrivelsen av parameterne, endepunkter og respons...
+URL-er til API-et, beskrivelsen av parameterne, endepunkter og respons ligger i Open API spesifikasjonen på 
+SwaggerHub
+
+API-et for boligsameie har bare ett endepunkt: Innsending. Dette endepunktet tar i mot tredjepartsopplysninger for 
+boligsameier. Ett kall mot API-et er en rapportering for ett boligsameie gitt av en oppgavegiver og som gjelder ett 
+inntektsår. 
+
+API-et validerer mottatte data mot JSON schema beskrevet på SwaggerHub. Se [feilkoder](boligsameie?tab=Feilkoder) for 
+relaterte feilmeldinger.
+
+Se også [eksempler](boligsameie?tab=Eksempler) for de ulike endepunktene.
+
+### Duplikate kall mot API-et
+Erfaringsmessig forekommer det av og til at API-et kalles med identiske data (samme oppgavegiver, payload og 
+parameterverdier) flere ganger innenfor et kort tidsrom. Hvis API-et oppdager et slikt duplikat kall, svarer det med OK 
+uten å foreta seg noe mer. Dette gjøres for å forhindre unødvendig behandling av redundante kall. 
+For å bestemme om et API kall er et duplikat brukes blant annet idempotencyKey parameteret API-et ble kalt med. Hvis et 
+identisk kall blir gjort 24 timer eller senere enn originalkallet, blir det ikke regnet som et duplikat.
```

Comment:
Burde vi kanskje si noe om at idempotensnøkkelen må genereres for hver unike innsending?